### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by wrdg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,35 +1,21 @@
...
-<br>
-
-### Money / Donation
-
-All The Mods has never been about making money, we just love playing games and making cool modpacks and contributing to the modded community. We understand that sometimes people like to "say thanks" so by request we have setup a Patreon with some perks and are considering expanding some ATM branded merchandise as well. Any money that we do get is used to "give back" such as hosting free public ATM servers   
-
-<a href="https://www.patreon.com/allthemods">
-	<img src="https://c5.patreon.com/external/logo/become_a_patron_button.png" alt="Support us on Patreon"  width="200" height="47">
-</a>  
+[![Logo Image](https://i.imgur.com/R3RgVCP.png)](https://minecraft.curseforge.com/members/ATMTeam/projects)
```
Comment (martmists) [inline README.md]: Shouldn't the ATM3 logo be used rather than the AllTheMods title?

File: README.md [hunk 2]
```diff
@@ -1,35 +1,21 @@
...
 
+[![CurseForge](http://cf.way2muchnoise.eu/full_269708_downloads.svg)](https://minecraft.curseforge.com/projects/all-the-mods-3) [![Discord](https://img.shields.io/discord/254530689225981953.svg?&label=discord)](https://discordapp.com/invite/rbSZNDQ) [![Subreddit](https://img.shields.io/badge/subreddit-r%2Fallthemods-red.svg)](https://reddit.com/r/allthemods) [![GitHub Contributors](https://img.shields.io/github/contributors/AllTheMods/ATM-3.svg)](https://github.com/AllTheMods/ATM-3/graphs/contributors) [![GitHub Issues](https://img.shields.io/github/issues/AllTheMods/ATM-3.svg)](https://github.com/AllTheMods/ATM-3/issues)
+
+# All The Mods 3
+**All the Mods** started out as a private pack for just a few friends of mine that turned into something others wanted to play!
+
+It has all the basics that most other "big name" packs include but with a nice mix of some of newer or lesser-known mods as well.
```
Comment (martmists) [inline README.md]: `mix of some of newer` -> mix of some newer*

File: README.md [hunk 3]
```diff
@@ -1,35 +1,21 @@
...
+
+For example; Thaumcraft, Ender IO, Thermal Expansion for the big names and Rustic, Traverse and FastWorkBench for the not as well known!
+
+Can you get to the Creative items by making the "**ATM Star**"?
+
+In **All the Mods 3** we will continue the tradition adding many new mods while going for more stability.
```
Comment (martmists) [inline README.md]: `the tradition adding` -> the tradition **of** adding*

File: README.md [hunk 4]
```diff
@@ -1,35 +1,21 @@
...
+[![CurseForge](http://cf.way2muchnoise.eu/full_269708_downloads.svg)](https://minecraft.curseforge.com/projects/all-the-mods-3) [![Discord](https://img.shields.io/discord/254530689225981953.svg?&label=discord)](https://discordapp.com/invite/rbSZNDQ) [![Subreddit](https://img.shields.io/badge/subreddit-r%2Fallthemods-red.svg)](https://reddit.com/r/allthemods) [![GitHub Contributors](https://img.shields.io/github/contributors/AllTheMods/ATM-3.svg)](https://github.com/AllTheMods/ATM-3/graphs/contributors) [![GitHub Issues](https://img.shields.io/github/issues/AllTheMods/ATM-3.svg)](https://github.com/AllTheMods/ATM-3/issues)
+
+# All The Mods 3
+**All the Mods** started out as a private pack for just a few friends of mine that turned into something others wanted to play!
+
+It has all the basics that most other "big name" packs include but with a nice mix of some of newer or lesser-known mods as well.
+
+For example; Thaumcraft, Ender IO, Thermal Expansion for the big names and Rustic, Traverse and FastWorkBench for the not as well known!
+
+Can you get to the Creative items by making the "**ATM Star**"?
```
Comment (martmists) [inline README.md]: Why is ATM Star in quotes?

File: README.md [hunk 5]
```diff
@@ -1,35 +1,21 @@
...
-
-<a href="https://www.patreon.com/allthemods">
-	<img src="https://c5.patreon.com/external/logo/become_a_patron_button.png" alt="Support us on Patreon"  width="200" height="47">
-</a>  
+[![Logo Image](https://i.imgur.com/R3RgVCP.png)](https://minecraft.curseforge.com/members/ATMTeam/projects)
 
+[![CurseForge](http://cf.way2muchnoise.eu/full_269708_downloads.svg)](https://minecraft.curseforge.com/projects/all-the-mods-3) [![Discord](https://img.shields.io/discord/254530689225981953.svg?&label=discord)](https://discordapp.com/invite/rbSZNDQ) [![Subreddit](https://img.shields.io/badge/subreddit-r%2Fallthemods-red.svg)](https://reddit.com/r/allthemods) [![GitHub Contributors](https://img.shields.io/github/contributors/AllTheMods/ATM-3.svg)](https://github.com/AllTheMods/ATM-3/graphs/contributors) [![GitHub Issues](https://img.shields.io/github/issues/AllTheMods/ATM-3.svg)](https://github.com/AllTheMods/ATM-3/issues)
```
Comment (martmists) [inline README.md]: This needs some cleaning up

File: README.md [hunk 6]
```diff
@@ -1,35 +1,21 @@
-# ATM-3
-
-All The Mods 1.12
-======
-This is the official repository and issue-tracker for All The Mods 3 1.12
-  
-Does "All The Mods" *really* contain ALL THE MODS? No, of course not.
-  
-#### Curse Links:
```
Comment (martmists) [inline README.md]: What's the reason for removing the links to the other modpacks?

File: README.md [hunk 7]
```diff
@@ -1,35 +1,21 @@
...
+
+It has all the basics that most other "big name" packs include but with a nice mix of some of newer or lesser-known mods as well.
+
+For example; Thaumcraft, Ender IO, Thermal Expansion for the big names and Rustic, Traverse and FastWorkBench for the not as well known!
+
+Can you get to the Creative items by making the "**ATM Star**"?
+
+In **All the Mods 3** we will continue the tradition adding many new mods while going for more stability.
+
+**Does "All The Mods" really contain ALL THE MODS? No, of course not:** *[Modlist](https://rawgit.com/AllTheMods/ATM-3/master/modlist.html)*
```
Comment (martmists) [inline README.md]: `No, of course not: Modlist` should be on a new line and not in bold

File: README.md (hunk 6 above)
Comment (wrdg) [reply README.md]: The projects are part of an organization they can be found under the same creator. If you really feel like updating every readme advertising all of the other modpacks, would seem tedious. I feel like each pack should have it’s own tailored readme.

File: README.md (hunk 5 above)
Comment (wrdg) [reply README.md]: What needs to be cleaned? They’re inline under the logo.

File: README.md (hunk 4 above)
Comment (wrdg) [reply README.md]: You’ll notice that all the descriptions were ripped off of CurseForge.

File: README.md (hunk 7 above)
Comment (wrdg) [reply README.md]: It is on a new line, also again the descriptions were ripped from CurseForge.

File: README.md (hunk 1 above)
Comment (wrdg) [reply README.md]: I figured ATM is a team, and should have it's logo representing the pack.

Comment (martmists) [reply README.md]: image
Uncentered, messy colors

File: README.md (hunk 7 above)
Comment (martmists) [reply README.md]: This is still not resolved

File: README.md (hunk 1 above)
Comment (martmists) [reply README.md]: whoops, meant to reply this to the comment on the badges

File: README.md (hunk 7 above)
Comment (martmists) [reply README.md]: image

Comment (wrdg) [reply README.md]: Not a big fan of how it looks on a new line, but ok. It's not listing nor is a new paragraph of information.

File: README.md (hunk 1 above)
Comment (wrdg) [reply README.md]: Honestly centering the image would be a easier if it was a scalable vector, do you guys have an actual location where I can get all the assets for ATM? I removed the contributors badge and changed the color for the discord badge, it's less "messy" colors now and match the header image, also is now centered with the current size of the header image. 

If you can provide more constructive feedback on the changes I make in the future, this process would be much easier for me to understand what you would like changed. Vague critic without a providing a recommendation to your idea of a change makes it difficult to understand you idea of what you want.

Comment (martmists) [reply README.md]: To center it, can't you simply use <center> ... </center>? If not, you could inline CSS it to make it centered.

I'll check the badges later.

Comment (wrdg) [reply README.md]: Never was a fan of using CSS and/or HTML in a base markdown file, doesn’t look clean and looks extremely out of place.

**Comment by OrdinatorStouff** [comment]
Why was this closed?

**Pull request closed without merging**